docs/book/src/developer/tilt.md

```diff
@@ -297,6 +299,22 @@ kustomize_substitutions:
   WORKER_MACHINE_COUNT: 3
 ```
 
+### Cleaning up your kind cluster and development environment
+
+After stopping Tilt, you can clean up your kind cluster and development environment by running
+
+```bash
+make kind-reset
+```
```

Comment:
q: should this be part of make clean (like all the clean targets)?

Reply:
I can see why they make sense as 2 different targets. `make clean` is to delete all auto-generated files that are now part of the project folder strucutre. `make kind-reset` doesn't really fall under the same category.

Reply:
In that case do we still want to rename `make kind-reset` to `make clean-tilt`? I had it next to `make kind-cluster` in the file as it's the inverse operation.

Reply:
> In that case do we still want to rename make kind-reset to make clean-tilt?

We should document how to clean up the kind cluster (using `make kind-reset`) and clean up the generated files with `make clean`. The current documentation looks good. I dont think we need any change here.

> I had it next to make kind-cluster in the file as it's the inverse operation.

Having `make kind-reset` next to `make kind-cluster` makes sense, since as you said they it is the inverse operation.